src/test/java/org/springframework/integration/kafka/rule/KafkaEmbedded.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.springframework.integration.kafka.rule;
+
+import static scala.collection.JavaConversions.asScalaBuffer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import com.gs.collections.api.block.function.Function;
+import com.gs.collections.impl.list.mutable.FastList;
+import com.gs.collections.impl.utility.ListIterate;
+import kafka.Kafka;
+import kafka.server.KafkaConfig;
+import kafka.server.KafkaServer;
+import kafka.utils.SystemTime$;
+import kafka.utils.TestUtils;
+import kafka.utils.TestZKUtils;
+import kafka.utils.Utils;
+import kafka.utils.ZKStringSerializer$;
+import kafka.zk.EmbeddedZookeeper;
+import org.I0Itec.zkclient.ZkClient;
+import org.I0Itec.zkclient.exception.ZkInterruptedException;
+import org.junit.rules.ExternalResource;
+import scala.collection.JavaConversions;
+
+import org.springframework.integration.kafka.core.BrokerAddress;
+
+/**
+ * @author Marius Bogoevici
+ */
+@SuppressWarnings("serial")
```

Comment:
Actually, it's because it uses some GS Collections code ... being a test class I just skipped annotating every compilation unit.